README.md

```diff
@@ -1083,6 +1120,15 @@ __AWESOME__ apps counter: __130__ 🎉
 - [x] [GitLab](https://gitlab.com/proninyaroslav/libretorrent)
 - [ ] Official page
 
+# TV Shows Tracker
+### Showly ❤️
```

Comment:
I heard about this app, but trackers scared me, so I do not know what to think about this.

Reply:
The dev knows that some users don't like those trackera and is working to reduce the offending in building the app for f-droid .
As I wrote it will soon ae added on f-droid.

 [Read more here](https://github.com/michaldrabik/showly-2.0/issues/46) and [here](https://github.com/michaldrabik/showly-2.0/issues/537)